tests/test_ping_plugin.py

```
@@ -27,219 +27,147 @@
     @patch("mmrelay.meshtastic_utils.connect_meshtastic")
     @patch("asyncio.sleep")
-    def test_handle_meshtastic_message_ping_direct_message(
-        self, mock_sleep, mock_connect
-    ):
-        """
-        Test that the plugin correctly handles a "ping" message received as a direct Meshtastic message.
-
-        Verifies that the plugin checks channel enablement for direct messages, sends a direct "pong" response to the sender, and returns True to indicate the message was handled.
-        """
-        # Mock meshtastic client
+    def test_explicit_ping_direct_message(self, mock_sleep, mock_connect):
```

Ruff (0.15.9)

[warning] Unused method argument: `mock_sleep`

(ARG002)